css/_uswds-theme-custom-styles.scss

```diff
@@ -1433,6 +1433,18 @@ code[class*="language-plaintext"] {
   @include external-link;
 }
 
+// External link - Safari only fix
+@media not all and (min-resolution:.001dpcm) { 
+  @supports (-webkit-appearance:none) and (stroke-color:transparent) {
+    #{$external-href}::after {
+      display: inline-block;
+    }
+  }
+}
```

Comment:
Let's create an issue for this bug on USWDS.

A possible solution would be to use `background-size: cover` or `mask-size: cover` to make sure Safari renders it properly. This will require a change to how we set the sizing.

Reply:
Removed the safari-only code. Will create an issue to look at the Safari display issues.